src/mscorlib/src/System/Text/DecoderBestFitFallback.cs

```diff
@@ -34,13 +31,7 @@ public override DecoderFallbackBuffer CreateFallbackBuffer()
         }
 
         // Maximum number of characters that this instance of this fallback could return
-        public override int MaxCharCount
-        {
-            get
-            {
-                return 1;
-            }
-        }
+        public override int MaxCharCount => 1;
```

Comment:
It cannot be just copy&paste unfortunately. CoreRT version is missing the .NET Standard 2.0 work; and the CoreCLR version is missing most of the cleanup.

@danmosemsft Do you plan to have somebody start the work to unify the two?